CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+
+#### Table Of Contents
+
+
+[Code of Conduct](#code-of-conduct)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Vulnerabilities](#reporting-vulnerabilities)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+  * [Documentation Maintenance](#documentation-maintenance)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Google C++ Styleguide](https://google.github.io/styleguide/cppguide.html)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+[Contribution Guide Thanks](#contribution-guide-thanks)
+
+## Code of Conduct
+
+1. Be nice and friendly
+2. Avoid any kind of abuses or strong language.
+3. Everything that You want to deliver to our community should be in well structured and polite form.
+4. Always make clear build before Pull Request
```

Comment:
IMHO, PR stuff is not something need to be defined in **Code of Conduct**

Reply:
I agree. Fixed.